test/e2e/data/infrastructure-vsphere/main/install-on-bootstrap/inject-install-on-bootstrap.yaml

```diff
@@ -61,9 +61,10 @@
           export DEBIAN_FRONTEND=noninteractive
           # sometimes the network is not immediately available, so we have to retry the apt-get update
           retry 10 5 "apt-get update"
-          apt-get install -y apt-transport-https ca-certificates curl
-          curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
-          echo 'deb https://apt.kubernetes.io/ kubernetes-xenial main' >/etc/apt/sources.list.d/kubernetes.list
+          apt-get install -y apt-transport-https ca-certificates curl gpg
+          mkdir -p -m 755 /etc/apt/keyrings
+          curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.28/deb/Release.key | gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
```

Comment:
Is that 1.28 there versioned by Kubernetes release?

Reply:
oh 🤔